New Alien Species: the Jibauni (#10867)
endless-sky/endless-sky: data/jibauni/jibauni culture conversations.txt

			choice
				`	"Sure."`
				`	"If you want."`
				`	"No thanks."`
					goto nothanksship
			`	"What is the name of your ship, <first>?"`
			choice
				`	"<ship>."`
				`	"Wait. How do you know my name?"`
					goto namequestion
				`	"I thought you would know that I am Captain <first> <last> of the <ship>."`
					goto thoughtyouknew
			label toldship
			`	"Okay. You probably already can see mine," the Jibauni winks at you.`
				goto whoyou
			label namequestion
			`	"You are the only living non-Jibauni anywhere in our space," the Jibauni reminds you. "Everyone knows who you are. Will you answer my question?"`
			choice
				`	"<ship>."`
					goto toldship
				`	"I thought you would know that I am Captain <first> <last> of the <ship>."`
			label thoughtyouknew
			`	"Nope," the Jibauni replies merrily. "We common Jibauni only know your first and last names."`
			label whoyou
			`	"What is your name?" you ask. "I am surprised you would want to know," is the answer. "But I am Ru-win, daughter of Alson and Weni."`
			choice
				`	"Thanks. Well, I need to get going."`
					goto finish
				`	"That's a nice name, what does it mean?"`

> 			`	"Ah, my name means 'The pilot'. A very imaginative name, isn't it? The 'Ru' means 'pilot', and names ending in consonants are traditionally used for female Jibauni. Traditional male names usually end in vowels. At any rate, I have some business to attend to, so goodbye."`

Reviewer: Optionally, you could add something here to track that First Last learned how traditional Jibuani names work. That way, in future missions when other Jibuani introduce themselves, you could add a small branch where you could acknowledge or ask about their names using what was learned here.
```suggestion
			`	"Ah, my name means 'The pilot'. A very imaginative name, isn't it? The 'Ru' means 'pilot', and names ending in consonants are traditionally used for female Jibauni. Traditional male names usually end in vowels. At any rate, I have some business to attend to, so goodbye."
			action
				set "Jibuani knowledge: traditional name structure"`
```

Author: ```suggestion
			`	"Ah, my name means 'The pilot'. A very imaginative name, isn't it? The 'Ru' means 'pilot', and names ending in consonants are traditionally used for female Jibauni. Traditional male names usually end in vowels. At any rate, I have some business to attend to, so goodbye."`
			action
				set "jibauni knowledge: name endings"
```

				decline

mission "Jibauni Culture: Rebel Report"
	minor
	source
		government "Jibauni"
	to offer
		has "language: Jibauni"
		random < 2
	on offer
		conversation
			`As you are passing by a restaurant, the door opens, and a small group of Jibauni come out. They are talking about politics, and as you listen, you realize that they are Rebel Jibauni acting as spies.`
			choice
				`	(Find a police station to report them at.)`
				`	(Confront them with what you heard.)`
					goto confront
			action
				payment 5000
			`	After a long search, you find a small police station tucked into the side of a hill. After a brief debate, you are admitted to the chief's office, where you tell them what just happened. A force is immediately dispatched to disarm and capture the Rebels. When it turns out that your story was correct, the chief thanks you profusely and gives you a reward of <payment>.`
				decline
			label confront
			action
				payment -2000
			`	The strangers act as if they don't know what you are talking about, but after leading you to a dark alley one of them pulls out a taser and zaps you.`

> 			`	When you wake up, you find that the spies have robbed you and left a message, which says to not be so foolhardy next time. In addition, they seemingly have left the planet, so telling the police will be useless.`

Reviewer: ```suggestion
			`	When you wake up, you find that the spies have robbed you and left a message, which says to not be so foolhardy next time. In addition, they seemingly have left the planet, so telling the police would be useless.`
```

				decline

mission "Rebel Jibauni Culture: Rude Rebel"
	minor
	source
		government "Jibauni (Rebel)"
	to offer
		has "language: Jibauni"
		random < 6
	on offer
		conversation

> 			`The part of the spaceport you are currently in seems to be a slum, although some architectural aspects are fascinating. There are many armed Rebels standing around, and, compared to the relative peace of the other Jibauni faction's planets, it is strange to see such a blatant display of weaponry.`

Reviewer: ```suggestion
			`This part of the spaceport seems to be a slum, although some architectural aspects are fascinating. Many armed Rebels stand around, and in contrast to the relative peace of the other Jibauni faction's planets, this blatant display of weaponry feels jarring.`
```
Made it a bit more concise with more emphasis on what I think you're going for.

				goto talkbattle
			label pretalkbattle
			`	"I think I am right, oh alien, so what does your opinion matter?"`
			label talkbattle
			choice
				`	"Well, you are rude, and nothing in the universe can deny it."`
				`	(Quietly walk away.)`
					goto walkedaway
			`	"Perhaps not, but I do deny it, and care nothing for any of your thoughts." The Jibauni turns its back on you, another rude gesture in Jibauni culture.`
			choice
				`	"Nor do I for yours."`
				`	(Quietly leave.)`
					goto walkedaway
				`	(Shoot the rude Rebel and leave.)`
					goto shot
				`	"Nor do I. Get out of here."`
					goto getout
			`	"Well, who cares," the Jibauni insults you again, while keeping its back turned.`
			choice
				`	"Leave immediately."`
				`	(Quietly leave.)`
					goto walkedaway
				`	(Shoot the rude Rebel and leave.)`
					goto shot
			label getout
			`	The Jibauni whips around to face you. "No. You do."`
			`	A squad of Rebel Jibauni soldiers come around a corner and forcibly escort you to your ship, then tell you that if you don't leave they will open fire.`
				flee
			label shot
			action
				"reputation: Jibauni (Rebel)" -= 1

> 			`	Your shot hits the Rebel squarely in the back, killing it instaneously. You get back to the <ship> without anyone seeing you, and launch immediately.`

Reviewer: ```suggestion
			`	Your shot hits the Rebel squarely in the back, killing it instantly. You get back to the <ship> without anyone seeing you, and launch immediately.`
```
